wagtaildraftail/client/wagtaildraftail.js

```diff
@@ -35,9 +45,9 @@ const initDraftailEditor = (fieldName, options = {}) => {
     // eslint-disable-next-line no-param-reassign
     options.entityTypes = options.entityTypes.map(entity => Object.assign(entity, {
       // TODO: Rename keys accordingly once changed in draftail https://github.com/springload/draftail/issues/49
-      control: sources[entity.source],
-      strategy: getEntityStrategy(entity.type),
-      component: decorators[entity.decorator],
+      control: getSource(entity.source),
+      strategy: getStrategy(entity.type) || getDefaultStrategy(entity.type),
+      component: getDecorator(entity.decorator),
```

Comment:
I disagree with using `window.wagtaildraftail.getSource` in the project's code. Sure it will make it easier for a person reading the source code that it's using the exact same function that the one they use in their custom components, but it would make the whole thing less readable and very verbose for a very limited number of people (who go through the source code rather than the documentation).

No objection to fallbacks.